index.js

```diff
@@ -208,4 +208,13 @@ projects.forEach((project, idx) =>  {
     projectInfo.append(projectTitle, projectStacks, buttonWrapper);
     projectCard.appendChild(projectInfo);
     projectsWrapper.appendChild(projectCard);
-})
\ No newline at end of file
+})
+
+const myForm = document.forms[0];
+myForm.addEventListener("submit", (event) => {
+    let emailInput = myForm.elements.namedItem("email");
+    if(emailInput.value !== emailInput.value.toLowerCase()){
+        emailInput.setCustomValidity("Expected only lower-case email address.");
```

Comment:
* Kindly show your error message to the user near the submit button as stated in the [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=If%20the%20validation%20is%20not%20OK%2C%20you%20show%20an%20error%20message%20to%20the%20user%20near%20the%20submit%20button%20informing%20them%20of%20the%20error%20and%20the%20form%20is%20not%20sent)
* When I put an email with Uppercase I get an error message but if I change it to a smaller case the error still comes up please fix this bug
![image](https://user-images.githubusercontent.com/52497246/190567510-9eb45f33-f853-4a74-88ef-9262376991b5.png)

Reply:
Kindly, I recommend you follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉

the email field should be of type "email" to validate in-build a valid email, in turn its code implemented in JS for validation is not working correctly (a form can be sent with any text)